script.js

```diff
@@ -0,0 +1,17 @@
+// dogInfo
+// addUp
+// #cart-total
+
+function dogInfo(name,breed,cost) {
+  alert('This is ' + name + ', they are a ' + breed + '. It is $' + cost + ' to bring them home with you!');
```

Comment:
Nice work! Consider using JS template literals for string concatenation. I think the syntax is nicer: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
This is great! Thanks!